lib/twirp/rails/engine.rb

```diff
@@ -41,16 +41,54 @@ def services
 
           @services = Twirp::Service.subclasses.map(&:new)
 
+          logging_hooks = {
+            before: proc { |rack_env, env|
+                      env[:request_start_time] = Time.current
+                      [rack_env, env]
+                    },
+            on_success: proc { |env|
+                          request_time = duration_in_ms(env[:request_start_time])
+                          ::Rails.logger.info("Twirp 200 in #{request_time}ms")
```

Comment:
Is 200 the only option?

Reply:
Yes, always success. 

Except, ah dammit, stop asking good questions! 🤣 

Our `ConditionalPost` will send back a 304 Not Modified, and that happens _after_ this proc. I'm tempted to log "Twirp Success" instead of "200"

Reply:
maybe the errors could be "Twirp permission_denied (403)" or something. Seems like having the HTTP status code is helpful for log searching, but dunno.